templates/web/base/waste/bulky/confirmation.html

```diff
@@ -0,0 +1,23 @@
+[% SET title = form.title ~%]
+[% PROCESS 'waste/header.html' %]
+
+<div class="govuk-panel govuk-panel--confirmation">
+    <h1 class="govuk-panel__title">
+        [% title %]
+    </h1>
+    <div class="govuk-panel__body">
+        <p>Your [% title | lower %];
+          [% IF report.user.email && report.get_extra_metadata('contributed_as') != 'anonymous_user' %]
+            a copy has been sent to your email address, [% report.user.email %].
+          [% END %]
+        </p>
+        <p>
+          [% IF report.category == 'Request new container' %]
```

Comment:
This presumably isn't needed at this point.

Reply:
Ah yes, good spot.